# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pesign/hotfix-no-mandoc.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pesign-115/src/Makefile.vanilla	2022-04-01 13:30:34.195837797 +0200
+++ pesign-115/src/Makefile	2022-04-01 13:30:55.786838911 +0200
@@ -9,7 +9,7 @@
 	   pesign-rpmbuild-helper pesign-authorize
 CFGTARGETS=tmpfiles.conf
 SVCTARGETS=pesign.sysvinit pesign.service
-MAN1TARGETS=authvar.1 efikeygen.1 pesigcheck.1 pesign-client.1 pesign.1
+MAN1TARGETS= # authvar.1 efikeygen.1 pesigcheck.1 pesign-client.1 pesign.1
 
 TARGETS=$(BINTARGETS) $(SVCTARGETS) $(MAN1TARGETS) $(CFGTARGETS)
 
@@ -86,8 +86,8 @@
 	$(INSTALL) -m 755 pesigcheck $(INSTALLROOT)$(bindir)
 	$(INSTALL) -d -m 755 $(INSTALLROOT)/etc/popt.d/
 	$(INSTALL) -m 644 pesign.popt $(INSTALLROOT)/etc/popt.d/
-	$(INSTALL) -d -m 755 $(INSTALLROOT)$(mandir)man1/
-	$(INSTALL) -m 644 $(MAN1TARGETS) $(INSTALLROOT)$(mandir)man1/
+	#$(INSTALL) -d -m 755 $(INSTALLROOT)$(mandir)man1/
+	#$(INSTALL) -m 644 $(MAN1TARGETS) $(INSTALLROOT)$(mandir)man1/
 	$(INSTALL) -d -m 755 $(INSTALLROOT)/etc/rpm/
 	$(INSTALL) -m 644 macros.pesign $(INSTALLROOT)/etc/rpm/
 	$(INSTALL) -d -m 755 $(INSTALLROOT)$(libexecdir)/pesign/
--- pesign-115/Makefile.vanilla	2022-04-01 13:34:50.287851009 +0200
+++ pesign-115/Makefile	2022-04-01 13:35:05.584851798 +0200
@@ -10,8 +10,8 @@
 SUBDIRS := include libdpe src
 
 install :
-	$(INSTALL) -d -m 755 $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
-	$(INSTALL) -pm 644 COPYING $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
+	#$(INSTALL) -d -m 755 $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
+	#$(INSTALL) -pm 644 COPYING $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
 	@$(call descend)
 
 install_systemd install_sysvinit : install
